source/user-groups.html.twig

```diff
@@ -242,22 +242,22 @@ title: "User Groups"
                     <h3>Inactive or Unknown Status:</h3>
                     <ul>
                         <li>
-                            <a href="http://www.memphisjug.org" target="_blank">Mid-South/Memphis Java User Group</a>
```

Comment:
We should probably keep these URLs around but not link to them?

Reply:
Yeah, I think we should keep the URL and just not link to it.

Reply:
e.g. `Mid-South/Memphis Java User Group - http://www.memphisjug.org/` ?

Reply:
Yah: `Mid-South/Memphis Java User Group - http://www.memphisjug.org (Site no longer active)`